src/vs/platform/accessibility/common/accessibility.ts

```diff
@@ -55,11 +55,26 @@ export const IAccessibleNotificationService = createDecorator<IAccessibleNotific
  */
 export interface IAccessibleNotificationService {
 	readonly _serviceBrand: undefined;
-	notify(event: AccessibleNotificationEvent, userGesture?: boolean): void;
+	notify(event: AccessibleNotificationEvent, userGesture?: boolean, forceSound?: boolean, allowManyInParallel?: boolean): void;
+	notifyLineChanges(event: AccessibleNotificationEvent[]): void;
```

Comment:
Why is this `LineChanges`?
Don't understand what `notifyLineChanges([AccessibleNotificationEvent.Save])` would mean. Maybe that needs to be a different enum?